src/main/java/org/javamodularity/moduleplugin/extensions/ModularityExtensionImpl.java

```diff
@@ -0,0 +1,97 @@
+package org.javamodularity.moduleplugin.extensions;
+
+import org.gradle.api.JavaVersion;
+import org.gradle.api.Project;
+import org.gradle.api.plugins.JavaPlugin;
+import org.gradle.api.tasks.compile.JavaCompile;
+import org.javamodularity.moduleplugin.JavaProjectHelper;
+
+import java.util.List;
+
+public class ModularityExtensionImpl implements ModularityExtension {
+
+    private final Project project;
+
+    public ModularityExtensionImpl(Project project) {
+        this.project = project;
+    }
+
+    //region JAVA RELEASE
+
+    //region STANDARD JAVA RELEASE
```

Comment:
What are these regions? Just markers of code that belongs together? If so, I think method names reflect that well enough, and should otherwise be broken into multiple classes.

Reply:
These are VisualStudio-style [custom folding region comments of IntelliJ IDEA](https://blog.jetbrains.com/idea/2012/03/custom-code-folding-regions-in-intellij-idea-111/). As you noticed, I use them for grouping code that belongs together so that I can fold the part I don't need to see right now.

But you're right that method names are quite clear in this case, and that otherwise, extracting classes might be cleaner as there's [some controversy about using regions](https://blog.codinghorror.com/the-problem-with-code-folding/).

To sum up, I'm absolutely fine with removing them 🙂